draft-ietf-quic-manageability.md

```diff
@@ -94,6 +94,12 @@ assumptions that the QUIC protocol design takes toward the expected network
 treatment. It also discusses how common network management practices will be
 impacted by QUIC.
 
+Since QUIC's wire image {{?WIRE-IMAGE=RFC8546}} is integrity protected and not
+modifiable on path, in-network operations are not possible without terminating
+the QUIC connection, for instance using a back-to-back proxy. Proxy operations
+are not in scope for this document, as QUIC proxies must be fully-fledged QUIC
+endpoints.
```

Comment:
Maybe add another half sentence saying something like "and therefore should follow guidance provided in the QUIC base specs {{...}} and the applicability statement {{...}}"...?

Reply:
yep